plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_aws_launcher.py

```diff
@@ -8,21 +8,16 @@
 from hydra.types import TaskFunction
 from omegaconf import DictConfig
 
-from hydra_plugins.hydra_ray_launcher.conf import (
-    RayClusterConf,
-    RayInitConf,
-    RayRemoteConf,
-    RsyncConf,
-)
+from hydra_plugins.hydra_ray_launcher.conf import RayClusterConf, RsyncConf
 
 log = logging.getLogger(__name__)
 
 
 class RayAWSLauncher(Launcher):
     def __init__(
         self,
-        ray_init_cfg: RayInitConf,
-        ray_remote_cfg: RayRemoteConf,
+        ray_init_cfg: DictConfig,
+        ray_remote_cfg: DictConfig,
```

Comment:
why?

Reply:
I'm not exactly sure what `why?` means here. Are you referring to changing them to `DictConfig` from `RayRemoteConf `? If so, that's because I removed the `RayRemoteConf ` class.